### PR TITLE
Add example pods to demonstrate CPU manager.

### PR DESCRIPTION
**Pull request by ConnorDoyle**
**What this PR does / why we need it**:

Adds example pods and README for the CPU manager static policy which is a new feature in Kubernetes v1.8.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: examples/cpu-manager/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+```
+$ watch ./examples/cpu-manager/pod-ips
+exclusive-1 http://172.17.0.4
+shared http://172.17.0.3
+```
+
+![Sample cpuset-visualizer output](https://user-images.githubusercontent.com/379372/28648573-974693ce-7223-11e7-84ed-17cce11910ff.png)
+
+_Sample cpuset-visualizer output._
```
Comment (brendandburns) [inline examples/cpu-manager/README.md]: Did you mean to fill this in?

File: examples/cpu-manager/pod-ips [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors.
```
Comment (brendandburns) [inline examples/cpu-manager/pod-ips]: 2017

**Comment by brendandburns** [comment]
Generally LGTM, small comments.

File: examples/cpu-manager/README.md (hunk 1 above)
Comment (ConnorDoyle) [reply examples/cpu-manager/README.md]: It's supposed to be a desc for the image just above. Better way to format this?

File: examples/cpu-manager/pod-ips (hunk 2 above)
Comment (ConnorDoyle) [reply examples/cpu-manager/pod-ips]: Thanks, will fix

File: examples/cpu-manager/README.md (hunk 1 above)
Comment (ConnorDoyle) [reply examples/cpu-manager/README.md]: Think it will read better above the image. Will move it up.

**Comment by ConnorDoyle** [comment]
@brendanburns thanks for speedy review; updated

**Comment by ConnorDoyle** [comment]
@brendanburns friendly nudge

**Comment by ConnorDoyle** [comment]
Could we add this to the v1.8 milestone so it can enter the submit queue?

**Comment by sjenning** [comment]
pull-kubernetes-federation-e2e-gce blocked by https://github.com/kubernetes/kubernetes/issues/52270

**Comment by ConnorDoyle** [comment]
xref https://github.com/kubernetes/features/issues/375

**Comment by sjenning** [comment]
@derekwaynecarr can we get this in the 1.8 milestone?  Just examples to go with CPU manager all under `examples/cpu-manager`.

/test pull-kubernetes-kubemark-e2e-gce

**Pull request merged**